### DISCUSSION TITLE
Expose an API for StorageProvider to get jobs updated between intervals

### DISCUSSION THREAD

**Discussion by jhchee** [jobrunr/jobrunr#725] [Ideas]
Currently, the StorageProvider exposes an API to get jobs updated before an instant.
```java
List<Job> getJobs(StateName state, Instant updatedBefore, PageRequest pageRequest);
```
Potentially, we can have an API that get jobs updated between intervals.

```java
List<Job> getJobsBetween(Instant start, Instant end, PageRequest pageRequest);
```
This enables an use case for summarizing jobs executed between intervals e.g. daily report on success/failure rate.

**Comment by rdehuyss** [comment]
That's possible in the Pro version 😉.
